devops/scripts/boot-strap-venv.sh

```diff
@@ -51,7 +51,7 @@ function virtualenv_bootstrap() {
         then
             p=$(command -v "python${PYTHON_VERSION}" 2> /dev/null || command -v python3)
             echo "Creating ${p} virtualenv in ${VENV}"
-            virtualenv -p "${p}" "${VENV}"
```

Comment:
This is breaking staging builds, which apparently have `virtualenv` but not `python3-venv`!! https://app.circleci.com/pipelines/github/freedomofpress/securedrop/7484/workflows/13f7df38-3382-4444-906f-5868e6deac89/jobs/78739

Will put in a fix in the staging PR.